File: backend/src/main/java/com/example/Flicktionary/domain/movie/repository/MovieCastRepository.java
package com.example.Flicktionary.domain.movie.repository;

import com.example.Flicktionary.domain.movie.entity.MovieCast;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface MovieCastRepository extends JpaRepository<MovieCast, Long> {

    List<MovieCast> findMoviesByActorId(Long actorId);
}

File: backend/src/main/kotlin/com/example/Flicktionary/domain/movie/repository/MovieCastRepository.kt
package com.example.Flicktionary.domain.movie.repository

import com.example.Flicktionary.domain.movie.entity.MovieCast
import org.springframework.data.jpa.repository.JpaRepository

interface MovieCastRepository : JpaRepository<MovieCast, Long> {
    fun findMoviesByActorId(actorId: Long): List<MovieCast>
}
